packages/gatsby-plugin-catch-links/README.md

```diff
@@ -32,7 +32,7 @@ Regular expression for paths to be excluded from being handled by this plugin.
 {
   resolve: `gatsby-plugin-catch-links`,
   options: {
-    excludePattern: /(excluded-link|external)/,
+    excludeRegex: /(excluded-link|external)/,
```

Comment:
Per https://github.com/gatsbyjs/gatsby/pull/17334 - it probably should stay as is, unless there are more bugs with that?

Reply:
I was using latest and `excludePattern` was not being evaluated–only actually worked when I hanged to `excludeRegex`

Reply:
Hmmm, I just checked both and both works

Reply:
must've had a typo originally or something…both work fine for me now as well 🙄